code/script.js

```diff
@@ -14,7 +14,7 @@ let choiceFood = "Pizza";
 do {
   //Present menu options (Pizza, Pasta, Salad)
   foodChoiceNum = prompt("What would you like to eat?\nPlease type 1,2 or 3:\n(1) Pizza\n(2) Pasta\n(3) Salad");
-  // I found the \n for new line option before our Tuesday lesson
+  // I found the \n for new line option but not sure if it makes the code look messier than just starting a new line
```

Comment:
I felt like it made my code look messier, but looking at your code I personally prefer the way you use \n as it looks tidy.